site/en/docs/privacy-sandbox/topics/index.md

```diff
@@ -220,10 +220,11 @@ order.
 The Topics explainer proposes that each topic object in the array returned by
 `document.browsingTopics()` would have three properties:
 
--  `value`: a number identifying the topic in the taxonomy
+-  `configVersion`: a string identifying the current configuration
+-  `modelVersion`: a string identifying the machine-learning classifier used to infer site
 -  `taxonomyVersion`: a string identifying the set of topics currently in use by the browser
--  `classifierVersion`: a string identifying the machine-learning classifier used to infer site
-topics from hostnames
+-  `topic`: a number identifying the topic in the [taxonomy](#how-would-topics-be-curated-and-selected)
```

Comment:
```suggestion
-  `topic`: an integer identifying the topic in the [taxonomy](#how-would-topics-be-curated-and-selected)
```

Reply:
I assume that it must be a whole number? If so, please make this update!

Reply:
Good to be specific — it is indeed a (positive :) integer.